### ISSUE TITLE
Explore > Places links returning Failed to search (500 - Internal Server Error)

### ISSUE THREAD

**Issue by bestrocker221** [immich-app/immich#5726]
### The bug

After upgrading to latest version, the squared boxes showing the Places under Explore no longer work.
When clicking to a recognized place, it returns Failed to search (500 - Internal Server Error)

The logs from the console are:
```
immich_server [Nest] 7 .. ERROR [ErrorInterceptor] Failed to search
immich_server [Nest] 7 .. ERROR [ErrorInterceptor] Error: Missing query
```

### The OS that Immich Server is running on

debian 11

### Version of Immich Server

v 1.91.0

### Version of Immich Mobile App

v 1.91.0

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Same as latest upstream
```

### Your .env content

```Shell
Same as latest upstream
```

### Reproduction steps

```bash
1. upgrade from 1.90.2 to 1.91.0
2. no error shown on logs
3. Go to Explore > Places and click on one
...
```

**Comment by auanasgheps** [comment]
I have this issue as well.

**Comment by alextran1502** [comment]
Thank you, this previously used Typesense to search, but we haven't updated it to use the new mechanism yet. Will fix this in future releases

**Comment by 9k001** [comment]
There is something wrong with my deployment.
The previously used environment variables do not work.
For example:
DB_HOSTNAME.
DB_PROT.
DB_USERNAME.

Server end.
Microservices end.
The container log shows that the database was not found.
I am using an independently deployed PostgreSQL

**Comment by spupuz** [comment]
same problem fo me.

**Comment by pedropombeiro** [comment]
This problem was recognized above, please don't comment with "me too".

**Comment by alextran1502** [comment]
fixed in 1.91.1
